contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol

```diff
@@ -7,12 +7,14 @@ import { Lib_OVMCodec } from "../../libraries/codec/Lib_OVMCodec.sol";
 import { Lib_AddressResolver } from "../../libraries/resolver/Lib_AddressResolver.sol";
 import { Lib_MerkleUtils } from "../../libraries/utils/Lib_MerkleUtils.sol";
 import { Lib_RingBuffer, iRingBufferOverwriter } from "../../libraries/utils/Lib_RingBuffer.sol";
+import { Lib_SafeMathWrapper } from "../../libraries/wrappers/Lib_SafeMathWrapper.sol";
```

Comment:
Do we need this import for anything else?

Reply:
nope, you're right, it's dangling. I'll get rid of it!